### PR TITLE
optimize user image upload

### PR DESCRIPTION
**Pull request by blaysku**

### CODE REVIEW

File: resources/assets/js/components/user/MasterUser.vue [hunk 1]
```diff
@@ -431,6 +451,26 @@
                         })
                 })
             },
+            openPhotoModal(type) {
+                if (type == 'avatar')
+                    this.showAllAvatar = true
+                else
+                    this.showAllImage = true
+                if (!this.listPhoto.length) {
```
Comment (pham-van-huy) [inline resources/assets/js/components/user/MasterUser.vue]: convention

File: resources/assets/js/components/user/MasterUser.vue [hunk 2]
```diff
@@ -431,6 +451,26 @@
...
+                    this.getUserPhotos()
+                }
+            },
+            getUserPhotos() {
+                this.photoPage++
+                get(`user/${this.pageId}/get-photos-user?page=${this.photoPage}`)
+                    .then(res => {
+                        this.totalPage = res.data.photos.last_page
+                        this.listPhoto = this.listPhoto.concat(res.data.photos.data)
+                        if (this.photoPage == res.data.photos.last_page)
```
Comment (pham-van-huy) [inline resources/assets/js/components/user/MasterUser.vue]: đây nữa

File: resources/assets/js/components/user/MasterUser.vue [hunk 3]
```diff
@@ -303,9 +303,21 @@
...
+            pageType: 'user',
+            listPhoto: [],
+            photoPage: 0,
+            isLastPhotoPage: false
         }),
         mounted() {
+            $('.tab-content').scroll((e) => {
+                const el = $(e.currentTarget)
+                if (el.scrollTop() + el.innerHeight() + 1 >= el[0].scrollHeight) {
+                    if (!this.isLastPhotoPage)
```
Comment (MinhTri1911) [inline resources/assets/js/components/user/MasterUser.vue]: sao chú k để lên `el.scrollTop() + el.innerHeight() + 1 >= el[0].scrollHeight && !this.isLastPhotoPage` luôn ??

File: resources/assets/js/components/user/MasterUser.vue [hunk 4]
```diff
@@ -303,9 +303,21 @@
...
-            pageType: 'user'
+            pageType: 'user',
+            listPhoto: [],
+            photoPage: 0,
+            isLastPhotoPage: false
         }),
         mounted() {
+            $('.tab-content').scroll((e) => {
+                const el = $(e.currentTarget)
+                if (el.scrollTop() + el.innerHeight() + 1 >= el[0].scrollHeight) {
```
Comment (MinhTri1911) [inline resources/assets/js/components/user/MasterUser.vue]: convention

**Pull request merged**